lib/vmdb/configuration_encoder.rb

```diff
@@ -22,7 +22,7 @@ def self.symbolize!(h)
     def self.load(data, symbolize_keys = true, &block)
       return {} if data.blank?
 
-      if defined?(Rails) && !Rails.env.production?
+      if !defined?(Rails) || !Rails.env.production? || ENV['ERB_IN_CONFIG']
```

Comment:
For all of our apps, we will have `Rails` installed/defined, but yes that is a difference.

rails|production|env|load erb before?|after|notes
----|---------------|-----|--------|-----|---|---
Y|Y|Y|N|**Y**|new usecase
Y|Y|N|N|N|**common**
Y|N|*|Y|Y|**common**
N|*|*|N|*Y*|not reached in our code